### PR TITLE
Add new cheats spawn commands

### PR DESCRIPTION
**Pull request by 0lvin**
* Spawn entity by coordinates: `spawnentity`
* Spawn entity to start point: `spawnonstart`
* Add optional animation for player model preview.

### CODE REVIEW

File: src/game/g_cmds.c [hunk 1]
```diff
@@ -1276,7 +1276,116 @@ Cmd_Teleport_f(edict_t *ent)
...
+		ent->s.angles[1] = atof(gi.argv(6));
+		ent->s.angles[2] = atof(gi.argv(7));
+	}
+	// flags
+	if (gi.argc() >= 9)
+	{
+		ent->spawnflags = atoi(gi.argv(8));
+	}
+
+	ent->classname = strdup(gi.argv(1));
```
Comment (devnexen) [inline src/game/g_cmds.c:1323]: these new cases might need a specific handling because of this ?

Comment (0lvin) [reply src/game/g_cmds.c:1323]: @devnexen Could you please explain little bit more? What do you mean?

Comment (devnexen) [reply src/game/g_cmds.c:1323]: sorry for lack of clarity but do you need to free those ?

Comment (0lvin) [reply src/game/g_cmds.c]: i see, thank you.

Should be fixed in 12aeb316bbd550c77b51158c001a2659b0a98dcf 
```c
ent->classname = strdup(gi.argv(1));
```
vs
```c
ent->classname = gi.TagMalloc(strlen(gi.argv(1)) + 1, TAG_LEVEL);
strcpy(ent->classname, gi.argv(1));
```

File: src/game/g_cmds.c [hunk 2]
```diff
@@ -1276,7 +1276,118 @@ Cmd_Teleport_f(edict_t *ent)
...
+		ent->s.angles[1] = atof(gi.argv(6));
+		ent->s.angles[2] = atof(gi.argv(7));
+	}
+	// flags
+	if (gi.argc() >= 9)
+	{
+		ent->spawnflags = atoi(gi.argv(8));
+	}
+
+	ent->classname = gi.TagMalloc(strlen(gi.argv(1)) + 1, TAG_LEVEL);
```
Comment (devnexen) [inline src/game/g_cmds.c:1323]: I just noticed, but possibly G_CopyString could be slightly shorter :)

Comment (0lvin) [reply src/game/g_cmds.c]: Thank you, fixed.

**Review by devnexen — APPROVED** [review]

**Comment by Yamagi** [comment]
Is everyone fine with it and can I merge it?

**Comment by 0lvin** [comment]
It will be great :-)

**Comment by Yamagi** [comment]
Okay :)

**Pull request merged**